### PR TITLE
clang-tidy check for FWCore

### PR DESCRIPTION
**Pull request by davidlange6**
PR to apply clang-tidy checks to all FWCore files except those that are a part of open pull requests (as of an hour ago) and files in test directories

### CODE REVIEW

File: FWCore/FWLite/src/branchToClass.cc [hunk 1]
```diff
@@ -27,14 +27,14 @@ class BranchToClass : private TVirtualBranchBrowsable
   
 private:
   ///NOTE: do not call this, it is only here because ROOT demands it
-  BranchToClass();
+  BranchToClass() = delete;
```
Comment (Dr15Jones) [inline FWCore/FWLite/src/branchToClass.cc]: From the comment, it seems the constructor is needed.

**Comment by Dr15Jones** [comment]
please test

**Comment by davidlange6** [comment]
the comment is from 2006... - can you reconstruct if this was fixed in root6? [things do compile/run/give the right answers]

**Comment by Dr15Jones** [comment]
+1

**Comment by Dr15Jones** [comment]
The BranchToClass class is in an anonymous namespace within a .cc file. It can never actually be seen by ROOT. So false alarm.

**Comment by davidlange6** [comment]
+1

**Pull request merged**